bin/build_images

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+# Set OpenShift's environment (you'll need to be logged in to an OpenShift
+# server)
+_set_openshift_env
+
+# Run ansible-playbook
+_docker_run ansible-playbook build_images.yml "$@"
```

Comment:
:medal_sports: I've missed this one.

Reply:
FYI you are not forced to use the whole docker command, I used the `bin/ansible-playbook` helper.